### PR TITLE
Include driver filter in sv count query

### PR DESCRIPTION
**Pull request by BasLee**
Filter structural variant count by custom driver annotations. 

Filtering by custom driver annotations was already possibly for mutation and CNA counts: this PR reuses this sql filter in the sv count query.

image
_PR adds some dummy custom driver annotations to es_0_

Changes:

- modify mybatis query by including `fromIncludeCustomAnnotationsSV` in `structuralVariantCounts`
- add tests to `AlterationMyBatisRepositoryTest`
  - add more  sv custom driver annotations to `testSql.sql`
  - fix failing counting tests due to changes in test data
- add custom driver annotations to es_0 structural variant data

### CODE REVIEW

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java
Comment (dippindots) [inline persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java:24]: Do we need this?

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepositoryTest.java
Comment (dippindots) [inline persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepositoryTest.java:57]: same here. I don't the it's being used anywhere. Did I missed something?

File: persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml
Comment (dippindots) [inline persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml:199]: Could we add some comments on this? I don't quite understand this (what's the purpose).

Comment (dippindots) [reply persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml:199]: I see, this is moved here from the original code.

**Review by dippindots — APPROVED** [review]
Looks good to me!

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java
Comment (BasLee) [reply persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/AlterationMyBatisRepositoryTest.java]: Fixed in [5d29809](https://github.com/cBioPortal/cbioportal/pull/10092/commits/5d298094d42bba02543f79a930e19ffc847b668a)

File: persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml
Comment (BasLee) [reply persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml:200]: Indeed, was existing code; I added a comment i [d010d01](https://github.com/cBioPortal/cbioportal/pull/10092/commits/d010d019d290b394c3b7e9a97ec15c806c6ca077)

**Review by dippindots — APPROVED** [review]
LGTM! Once we fixed the validator test, we can merge it.

**Comment by BasLee** [comment]
> LGTM! Once we fixed the validator test, we can merge it.

The validator is fixed in [363488b](https://github.com/cBioPortal/cbioportal/pull/10092/commits/363488bd557ef6af2581a3f2c87fba48d1e4e413)

Some of the failing e2e tests in the "ci/circleci: end_to_end_tests_localdb" are failing because of changes in study_es_0. These tests are fixed in the front end PR ([commit](https://github.com/cBioPortal/cbioportal-frontend/pull/4552/commits/292e4f3531cbed78731a266bbacd55b94e6ca126)).

**Pull request merged**
